### PR TITLE
Adds build times to other forms of windows (AKA more Fornite removal)

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request
Hot take but [Fortnite](https://www.youtube.com/watch?v=QqRLVFRe9AU) is actually kind of fun when your opponents can't turn around and shit out a five story mansion with a three car garage when you so much as breathe in their direction.

I'm labeling this a fix because goof for some reason didn't do this in his last anti-Fortnite PR and I can't imagine he intended to _not_ do it.

## Why It's Good For The Game
Similarly, being able to shit out windows in combat is probably bad for balance. All the windows take the same amount of time - 0.5s for directional, 2s for full-tile.

## Changelog
:cl: Vekter
balance: Adds a build time to window types which didn't have ones before.
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -83,8 +83,8 @@ GLOBAL_LIST_INIT(glass_recipes, list ( \
 	return ..()
 
 GLOBAL_LIST_INIT(pglass_recipes, list ( \
-	new/datum/stack_recipe("directional window", /obj/structure/window/plasma/unanchored, time = 0, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
-	new/datum/stack_recipe("fulltile window", /obj/structure/window/plasma/fulltile/unanchored, 2, time = 0, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("directional window", /obj/structure/window/plasma/unanchored, time = 0.5, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("fulltile window", /obj/structure/window/plasma/fulltile/unanchored, 2, time = 2, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
```
Comment (Pickle-Coding) [inline code/game/objects/items/stacks/sheets/glass.dm:87]: Use the SECONDS define.

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 2]
```diff
@@ -141,8 +141,8 @@ GLOBAL_LIST_INIT(pglass_recipes, list ( \
 GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 	new/datum/stack_recipe("windoor frame", /obj/structure/windoor_assembly, 5, time = 0, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
 	null, \
-	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/unanchored, time = 0, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
-	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 0, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/unanchored, time = 0.5, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 2, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
```
Comment (Pickle-Coding) [inline code/game/objects/items/stacks/sheets/glass.dm:145]: Use the SECONDS define.

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 3]
```diff
@@ -179,8 +179,8 @@ GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 	. += GLOB.reinforced_glass_recipes
 
 GLOBAL_LIST_INIT(prglass_recipes, list ( \
-	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/plasma/unanchored, time = 0, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
-	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/plasma/fulltile/unanchored, 2, time = 0, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/plasma/unanchored, time = 0.5, on_solid_ground = TRUE, check_direction = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/plasma/fulltile/unanchored, 2, time = 2, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
```
Comment (Pickle-Coding) [inline code/game/objects/items/stacks/sheets/glass.dm:183]: Use the SECONDS define.

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 4]
```diff
@@ -214,7 +214,7 @@ GLOBAL_LIST_INIT(prglass_recipes, list ( \
 	. += GLOB.prglass_recipes
 
 GLOBAL_LIST_INIT(titaniumglass_recipes, list(
-	new/datum/stack_recipe("shuttle window", /obj/structure/window/reinforced/shuttle/unanchored, 2, time = 0, on_solid_ground = TRUE, check_direction = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("shuttle window", /obj/structure/window/reinforced/shuttle/unanchored, 2, time = 0.5, on_solid_ground = TRUE, check_direction = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
```
Comment (Pickle-Coding) [inline code/game/objects/items/stacks/sheets/glass.dm:217]: Use the SECONDS define.

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 5]
```diff
@@ -243,7 +243,7 @@ GLOBAL_LIST_INIT(titaniumglass_recipes, list(
 	. += GLOB.titaniumglass_recipes
 
 GLOBAL_LIST_INIT(plastitaniumglass_recipes, list(
-	new/datum/stack_recipe("plastitanium window", /obj/structure/window/reinforced/plasma/plastitanium/unanchored, 2, time = 0, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("plastitanium window", /obj/structure/window/reinforced/plasma/plastitanium/unanchored, 2, time = 2, on_solid_ground = TRUE, is_fulltile = TRUE, category = CAT_WINDOWS), \
```
Comment (Pickle-Coding) [inline code/game/objects/items/stacks/sheets/glass.dm:246]: Use the SECONDS define.

**Comment by TheVekter** [comment]
Deciseconds are stupid

**Comment by moocowswag** [comment]
i was wondering why plastitanium windows were able to be instabuilt despite being so much stronger

**Comment by TheVekter** [comment]
wherein I hit the griddy over the dead body of inconsistent combat strategies

**Pull request merged**